Faculty_notes.md

```diff
@@ -0,0 +1,165 @@
+
+* **Estimated time:** 1.5 hours (participatory live-coding, not including GDD extension)
+* **Target audience:** 3rd to 4th year ecology or biology students
+* **Prerequisites:** familiarity with R and the [tidyverse](https://www.tidyverse.org/).
+
+## Learning objectives
+> - Work with and plot time series data
+> - Fit models to ecological time series data
+> - Explore phenological concepts 
+
+## Data description
+
+The data used in this module is a subset of NEON plant phenology data, downloaded from
+the NEON data tutorial 
+["Work With NEON's Plant Phenology Data"](https://www.neonscience.org/neon-plant-pheno-data-r).
+The data download link is [here](https://ndownloader.figshare.com/files/9012085).
+The three files used are `pheno/phe_perindividual.csv`, `pheno/phe_statusintensity.csv`, and 
+`temp/SAAT_30min.csv`.
+
+## Teaching challenges to address
+
+When this module was implemented as a participatory live-coding lesson, 
+we didn't include the data processing steps (everything before *Plotting and fitting the data*).
+We provided students with a direct link to the joined and filtered dataframe.
+If the processing steps are covered in a class setting, they may be confusing
+or slow for students to get through.
+
+Students struggled to follow along during the date conversion steps, which 
+caused some downstream issues when the date columns hadn't been properly converted
+to use as the x-axis for fitting a function. 
+
+If presenting this material or a version of it as a live-coding exercise, 
+be cautious about the speed at which the material can be covered and 
+be prepared for it to take longer than expected. 
+
+## Assessment strategy
+
+We assessed students' ability to fit an oscillatory model to data with the following assignment question. Advanced classes could also consider using a form of spectral decomposition to analyze periodic data.
+
+**The Canadian lynx population cycle**
+
+    The Canadian lynx experiences large periodic changes in its population 
+    size over a timescale of several years. This is thought to be driven by 
+    oscillations in the population size of the snowshoe hare, the primary food 
+    source for the lynx. Read more about the lynx population cycle on this 
+    [Northwest Territories website](https://www.enr.gov.nt.ca/en/services/lynx/lynx-snowshoe-hare-cycle).
+    
+    R has a built-in dataset called `lynx` which contains annual population 
+    measurements for the Canadian lynx as a time series. 
+
+(a) Plot `lynx` vs. time in years using either `ggplot` or `qplot`. 
+    Plot points (`geom_point`) and a connecting line (`geom_line`). 
+    Create a time series that starts at 0 and ends at the total number of 
+    years in the dataset (total years $= 1934-1821$).
+    By eye, estimate the time between peaks in the population. 
+
+```{r}
+# SOLUTION
+
+years = seq(0, 1934-1821, by = 1)
+qplot(x = years, y = lynx) +
+  geom_line()
+
+# optional: make data frame 
+df = data.frame(lynx)
```

Comment:
Might be worth changing this variable name to something like `lynx_df` - R tragically has a built in function called `df` and that's caused some namespace issues for me in the past